### PR TITLE
Allow non-nominal multiple currencies promos

### PR DESCRIPTION
**Pull request by ericreis**
#### What is the purpose of this pull request?
Allow non-nominal multiple currencies promos.

#### What problem is this solving?
[ch26437](https://app.clubhouse.io/vtex/story/26437/trade-policy-with-multiple-currencies)
image

#### How should this be manually tested?
1. Go to https://eric--storecomponents.myvtex.com/admin/promotions/new/
2. Try to create promotions with multiple currencies

#### Screenshots or example usage
n/a

### CODE REVIEW

File: react/PromotionPage.js [hunk 1]
```diff
@@ -215,6 +240,11 @@ class PromotionPage extends Component {
     const currencyCode =
       uniqueCurrencyCodes.length === 1 ? uniqueCurrencyCodes[0] : undefined
 
+    const multipleCurrenciesAlertLevel = restriction.restrictedSalesChannels
```
Comment (kevinch) [inline react/PromotionPage.js]: No chance of `restrictedSalesChannels` being undefined?

File: react/PromotionPage.js [hunk 2]
```diff
@@ -278,7 +306,12 @@ class PromotionPage extends Component {
                     }
                   )
                 },
-              }}>
+              }}
+              onClose={
+                multipleCurrenciesAlertLevel === 'warning' &&
```
Comment (kevinch) [inline react/PromotionPage.js]: This string should be a const since it's used multiple times.

File: react/utils/promotion/validation/index.js [hunk 3]
```diff
@@ -295,25 +295,25 @@ export function validateEffectsSection(effects, intl) {
 
   switch (effects.activeEffectType.value) {
     case 'price':
-      ({ price: effects.price, isValid } = validatePriceEffect(
+      ;({ price: effects.price, isValid } = validatePriceEffect(
```
Comment (kevinch) [inline react/utils/promotion/validation/index.js]: Lint?

Comment (ericreis) [reply react/utils/promotion/validation/index.js]: yes, don't know what is going on here

**Pull request merged**